app/src/main/res/layout/activity_main.xml

```diff
@@ -7,13 +7,179 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <View
+        android:id="@+id/id_my_profile_up"
+        android:layout_width="@dimen/layout_param"
+        android:layout_height="@dimen/layout_param"
+        android:background="@color/profile_background"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/id_my_profile_down" />
+
+    <View
+        android:id="@+id/id_my_profile_down"
+        android:layout_width="@dimen/layout_param"
```

Comment:
layout_param is very vague naming, can you rename it to something more specific please? Also, if your dimensions are 0dp then it is not necessary to move them to resources, I personally prefer to keep them in the xml

Reply:
I have fixed it